app/components/Review/Components.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+
+export const StyledTitleRow = styled('th')`
+  padding: 16px !important;
+  border-top: 1px solid rgba(0, 0, 0, 0.16);
+`;
+
+export const StyledH4 = styled('h4')`
+  margin: 0;
+`;
+
+export const StyledColLeft = styled('th')`
+  // Todo: workaround for Jest styled component theme prop error
```

Comment:
We can uncomment these as well since we are wrapping our tests with `<GlobalTheme/>` provider.